### ISSUE TITLE
Update content-server and/or payments-server config files for consistency

### ISSUE THREAD

**Issue by LZoog** [mozilla/fxa#2073]
Spin-off of https://github.com/mozilla/fxa/pull/2053

The configuration files of `fxa-content-server` and `fxa-payments-server` have very different schemas. For the sake of consistency and in order to more easily share files across these two servers, the configs should better match.

For example, the content server config references `fxaccount_url` for the same object in the payment server's `servers.auth.url`.

This would enable us to, at the very least, share CSP middleware, which will essentially be duplicated when #2053 lands.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-247)
┆Issue Number: FXA-247

**Comment by clouserw** [comment]
This needs to be clarified and broken down further before we can make progress on it.

**Comment by clouserw** [comment]
There is also some question about whether we should do this at all, it sounds like?

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Closing in favor of https://github.com/mozilla/fxa/pull/7821
